src/aurora/types.h

```diff
@@ -385,7 +385,7 @@ enum GameID {
 };
 
 enum ResourceType {
-	kResourceImage,  ///< A music resource.
+	kResourceImage,  ///< A image resource.
```

Comment:
Should be "An image", though. :P

Also, the commit message should say "Fix", not "Fixed".